src/printer.ts

```diff
@@ -244,9 +244,8 @@ export class PugPrinter {
 		const wrapAttributesPattern: string = options.pugWrapAttributesPattern;
 		this.wrapAttributesPattern = wrapAttributesPattern ? new RegExp(wrapAttributesPattern) : null;
 
-		const codeSingleQuote: boolean = !options.pugSingleQuote;
 		this.codeInterpolationOptions = {
-			singleQuote: codeSingleQuote,
+			singleQuote: options.pugSingleQuote ?? options.singleQuote,
```

Comment:
This fallback is unnecessary cause it is already managed by `convergeOptions` in `src/options/converge.ts#L17` called in `src/index.ts#L91` and that is passed to the constructor of `PugPrinter`.

Reply:
I did that for consistence since the other options in that object do the same thing.